### PR TITLE
Update API version to 2022-10

### LINKED ISSUE #611: Latest Shopify Python package does not support the latest Shopify API version
## Overview/summary
Hello Shopify! I am the engineer who's taking care of updating the Shopify API to the latest version (`2022-10`), on behalf of Channable. We currently use version `10.0.0` of the Shopify Python package, and I noticed that in the latest version of the package, `12.0.0`, the latest API version is not supported (version `12.0.0` support API version `2022-04`). 

Could you please update the Shopify Python package in order to support the latest available Shopify API version, i.e. `2022-10`?

## Motivation

Our customers expect to be able to use the latest available versions of platforms and channels, in this case Shopify, hence it is of paramount importance for us to keep meeting these standards. 

### Area

- [ ] Add any relevant `Area: <area>` labels to this issue

---

## Checklist

- [ ] I have described this enhancement in a way that is actionable (if possible)

### PR DESCRIPTION
**Pull request by Alchez**
Fixes #611 

### Checklist

- [x] I have updated the CHANGELOG (if applicable)
- [x] I have followed the [Shopify Python](https://github.com/Shopify/shopify_python) guide

### CODE REVIEW

**Review by andres-movl — APPROVED** [review]

**Comment by lmarangio** [comment]
Re-ping :)

**Comment by adelespinasse** [comment]
Who can help move this along? @dylanahsmith ? @mllemango ?

**Comment by lmarangio** [comment]
Guys in one month there is going to be version `2023-01`, would be nice to close this issue before next week!

**Comment by occamssafetyrazor** [comment]
Anyone trying to get their app approved by Shopify will be blocked due an outdated API version. So Shopify is blocking apps due to their own outdated repository:
_"Your app is using an [API version](https://shopify.dev/concepts/about-apis/versioning) that is about to be deprecated. Address this breaking change by the date indicated in the linked screenshot."_

**Comment by dylanahsmith** [comment]
> can we get this merged?

It is currently blocked by "Required statuses must pass before merging", which seems to be due to two issues:
- [ ] "CLA" status is expected, which seems to not match the "Contributor License Agreement (CLA) / cla" passing status, which I think might require repo setting changes
- [ ] The CI workflow wasn't being triggered for pull requests from forks (only for push events) so I opened https://github.com/Shopify/shopify_python_api/pull/624)
- [ ] https://github.com/Shopify/shopify_python_api/pull/624 revealed that CI is also failing on master now

File: CHANGELOG
Comment (VijayaLakshmi210800) [inline CHANGELOG:2]: when this will be available in released version

Comment (adelespinasse) [reply CHANGELOG:2]: It seems to be available now. Version 12.1.0 is on pypi as of a few hours ago.

Comment (VijayaLakshmi210800) [reply CHANGELOG:2]: Thanks its available now

**Pull request merged**
